### PR TITLE
Add tptp lark parser

### PR DESCRIPTION
**Pull request by inpefess**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by inpefess** [comment]
@conda-forge-admin, please ping team

File: recipes/tptp-lark-parser/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tptp-lark-parser-{{ version }}.tar.gz
+  sha256: f54129f19af4d34003d08538823cb0190611387affa7df0b55723fa0cb55163f
+
+build:
+  skip: true  # [py>=311 or py2k]
```
Comment (ocefpaf) [inline recipes/tptp-lark-parser/meta.yaml:13]: This one can probably be noarch.

```suggestion
  noarch: python
```

File: recipes/tptp-lark-parser/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+    - poetry-core >=1.0.0
+  run:
+    - importlib_resources  # [py<39]
```
Comment (ocefpaf) [inline recipes/tptp-lark-parser/meta.yaml:23]: It is fine to install `importlib_resources` in >=39, the standard lib will be used there and we can make the package noarch.

```suggestion
    - importlib_resources
```

**Comment by inpefess** [comment]
@ocefpaf :+1:

**Pull request merged**